### DISCUSSION TITLE
How do I use on linux?

### DISCUSSION THREAD

**Discussion by winbamstudios** [CosmosOS/Cosmos#2528] [Q&A]
I have Cosmos installed on Ubuntu 22.04, but how do I make a project for it?

**Comment by AnErrupTion** [comment]
You'll need to make a project from Windows or use an existing project.

**Reply by winbamstudios** [reply]
oh

**Reply by AsertCreator** [reply]
idk but dotnet cli should work

**Reply by selkij** [reply]
> You'll need to make a project from Windows or use an existing project.

absolutely not, just see the docs

**Reply by selkij** [reply]
you can just create a blank project and add the required nupkgs Cosmos.System2 and Cosmos.Build
